global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "5.0.100-preview.3.20216.6"
```

Comment:
Can we upgrade to something newer? 
Do we need tihs for local development too or only the CI?

Reply:
Updated to public preview4.
When the bug in public preview5 got fix, we will update to that version.
Created a tracking issue https://github.com/NuGet/Client.Engineering/issues/367